Sources/Profile-Export.php

```diff
@@ -569,7 +571,10 @@ function ($datatype) use ($txt)
 	if (isset($_SERVER['HTTP_RANGE']))
 	{
 		while (@ob_get_level() > 0)
+		{
 			@ob_end_clean();
+			header_remove('content-encoding');
```

Comment:
You shouldn't need to do this inside the loop.

Reply:
Fixed